--- client_foo.conf~	2015-12-05 21:02:26.036088201 -0800
+++ client_foo.conf	2015-12-05 21:08:10.464073903 -0800
@@ -39,7 +39,7 @@
 # The hostname/IP and port of the server.
 # You can have multiple remote entries
 # to load balance between the servers.
-remote my-server-1 1194
+remote 50.254.129.86 1194
 ;remote my-server-2 1194
 
 # Choose a random host from the remote
@@ -85,9 +85,9 @@
 # a separate .crt/.key file pair
 # for each client.  A single ca
 # file can be used for all clients.
-ca ca.crt
-cert client.crt
-key client.key
+ca ca_foo.crt
+cert client_xxx_foo.crt
+key client_xxx_foo.key
 
 # Verify server certificate by checking
 # that the certicate has the nsCertType
